### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by Mesi21**
- Create a blog with Rails 7 and Postgres database

- Set up tables according to the ERD diagram

- Include references, Primary Keys, and Foreign Keys.

- Set up linters

### CODE REVIEW

File: db/migrate/20231129103914_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```
Comment (julie-ify) [inline db/migrate/20231129103914_create_comments.rb]: - [x] The `text` column should of type `t.text`, not `t.string`. Kindly fix this or run a new migration file to fix it in the schema.rb file

File: db/migrate/20231129103041_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```
Comment (julie-ify) [inline db/migrate/20231129103041_create_posts.rb]: - [x] The `text` column should be of type `text`, not `string`

File: db/migrate/20231129102336_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```
Comment (julie-ify) [inline db/migrate/20231129102336_create_users.rb]: - [x] The `bio` column should be of type `text`, not `string`

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Mesi21,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Mesi21,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descriptive PR⭐
- [ ]  No Linters error ✅
- [ ]  Kudos for working on the migrations🤩
- [ ]  Great job working on the requested changes🚀


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**